src/ds3.h

```diff
@@ -18,7 +18,6 @@
 
 #include <stdint.h>
 #include <string.h>
-#include <curl/curl.h>
```

Comment:
I seem to remember that there was a reason why this was here.  I thought that the browser was using it.  You may want to try and compile the browser with this latest SDK and see if anything breaks.